src/main/kotlin/cacophony/codegen/patterns/cacophonyPatterns/ValuePatterns.kt

```diff
@@ -5,131 +5,136 @@ import cacophony.codegen.patterns.SlotFill
-        override val tree = lhsSlot eq rhsSlot
+object EqualsValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot eq rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
 
-    object NotEqualsPattern : ValuePattern, BinaryOpPattern() {
-        override val tree = lhsSlot neq rhsSlot
+object NotEqualsValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot neq rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
 
-    object LessPattern : ValuePattern, BinaryOpPattern() {
-        override val tree = lhsSlot lt rhsSlot
+object LessValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot lt rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
 
-    object GreaterPattern : ValuePattern, BinaryOpPattern() {
-        override val tree = lhsSlot gt rhsSlot
+object GreaterValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot gt rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
 
-    object LessEqualPattern : ValuePattern, BinaryOpPattern() {
-        override val tree = lhsSlot leq rhsSlot
+object LessEqualValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot leq rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
 
-    object GreaterEqualPattern : ValuePattern, BinaryOpPattern() {
-        override val tree = lhsSlot geq rhsSlot
+object GreaterEqualValuePattern : ValuePattern, BinaryOpPattern() {
+    override val tree = lhsSlot geq rhsSlot
 
-        override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
-            TODO("Not yet implemented")
-        }
+    override fun makeInstance(fill: SlotFill, destination: Register): List<Instruction> {
+        TODO("Not yet implemented")
     }
+}
```

Comment:
NIT: As above, I think that it would be nice to use the same order of these operators in both `ConditionPatterns.kt` and here